package.json

```diff
@@ -6,11 +6,18 @@
   "author": "Shahed Nasser",
   "license": "ISC",
   "scripts": {
+    "build": "parcel build index.html",
+    "dev": "parcel index.html --open && lite-server",
+    "start": "npm run build && npm run dev && npm run watch:all",
     "compile": "lessc src/sbuttons.less dist/sbuttons.css && npm run prettier && npm run clean-css",
     "watch": "concurrently \"less-watch-compiler src dist\" \"onchange -v dist/sbuttons.css -- npm run clean-css \" ",
     "prettier": "prettier --write \"src/**/*.less\" \"dist/sbuttons.css\" \"assets/**\" ",
+    "lite": "lite-server",
     "check-prettier": "prettier --check \"src/**/*.less\" \"dist/sbuttons.css\" ",
-    "clean-css": "cleancss -o dist/sbuttons.min.css dist/sbuttons.css"
+    "clean-css": "cleancss -o dist/sbuttons.min.css dist/sbuttons.css",
+    "watch:all": "parallelshell 'npm run watch:scss' 'npm run watch:less' 'npm run lite'",
+    "watch:scss": "onchange 'css/*.scss' --npm run scss",
+    "watch:less": "onchange 'css/*.less' --npm run less"
```

Comment:
The command for "watch:less" should be `onchange -v src/*.less -- npm run compile`

Reply:
Done